public/js/controllers/game.js

```diff
@@ -229,7 +255,8 @@ angular.module('mean.system')
   }
 
   $scope.searchUsers = function () {
-    $http.get(`/api/search/users/${$scope.email}`)
+    const link = `/api/search/users/${$scope.email}`;
+    dataFactory.searchUsers(link)
```

Comment:
Another option would be to pass in just the `email` as opposed to constructing the link here. The `searchUsers` service should handle figuring out what endpoint to search from.

Probably move the generation of the link to the service so you have something like this.

```javascript
dataFactory.searchUsers($scope.email)
```

Then in the service you have something similar to this.
```javascript
searchUsers: userEmail => {
    return $http.get(`/api/search/users/${userEmail}`)
```